nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,276 @@
+
+Additional Git providers can be added by using Nautobot's flexible plugin system.  
+
+|Name|Summary|Related Plugin|
+|:--|:--|:--|
+|Backup Configs|Backup configuration data.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+|Intended Configs|Stores the intended configurations, this grabs Nautobot data and runs through Jinja Templates.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+|Jinja Templates|Repository that holds Jinja templates to be used to generate intended configs.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+
+## Repository Details
+This table defines repository parameters that are required to establish a repository connection.
+
+|Field|Explanation|
+|:--|:--|
+|Name|User friendly name for the repo.|
+|Slug|Computer-friendly name for the repo. Auto-generated based on the `name` provided, but you can change it if you wish.|
+|Remote URL|The URL pointing to the Git repo. Current git url usage is limited to `http` or `https`.|
+|Branch|The branch in the Git repo to use. Defaults to `main`.|
+|Token|The token is a personal access token for the `username` provided.  For more information on generating a personal access token see the corresponding links below. **Token is the only option supported currently.**|
+|Username|The Git username that corresponds with the personal access token above. Note not required for GitHub, but is for GitLab.|
+|Provides|Resource type(s) provided by this Git repo.|
+
+- [GitHub Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
+- [GitLab Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
+- [Bitbucket Personal Access Token](https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html)
+
+
+## Using Git Data Sources
+This section will focus on examples and use the `user-guide` branch on the `demo-git-datasources` repo: `https://github.com/nautobot/demo-git-datasource/tree/user-guide`.
+
+### Export Templates
+
+[Export Templates](../models/extras/exporttemplate.md) allow a user to export Nautobot objects based on a custom template.  Export templates can change over time depending on the needs of a user.  Allowing export templates to reference a Git repo makes managing templates easier.
+
+A template can be used to put objects into a specific format for ingestion into another system, tool, or report.  It is possible that different templates are needed depending on specific users or teams.  This can lead to sprawl of export templates.  To keep accurate templates synced with Nautobot the Git Data Sources extensibility feature can be used.
+
+#### Add a Repository
+Navigate to the Data Sources Git integration. **Extensibility -> Git Repositories**.
+
+![Menu showing "Git Repositories" Item](./images/git-as-data-source/00-git-data-source_1.png)
```

Comment:
I don't think `00-git-data-source_1.png`, `00-git-data-source_2.png`, etc. was what @jedelman8 had in mind for the image names here - I'm guessing he meant `01-git-data-source.png`, `02-git-data-source.png`, etc.

Reply:
Let me know which is preferred.  I was assuming prepended number was for a versioning technique.  If I need to change them I will.

Reply:
@glennmatthews  @jathanism  
Fixed, sorry for the confusion.